lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -2033,6 +2014,16 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
   }
 #endif
 
+  void createSpecialStlibBuiltinTypes() {
```

Comment:
Can you add a comment here about what these types are and what makes them "special" and why we emit them into every CU?